devtools/common/src/test/resources/gradle-project/gradle.properties

```diff
@@ -1 +1 @@
-quarkusVersion = 999-SNAPSHOOT
\ No newline at end of file
+quarkusVersion = 0.21.1
```

Comment:
This is probably not a good question and I am assuming that I am missing something fundamental here, but how would this work with the current code if the test uses a specific version?